root/meta/CMakeLists.txt

```diff
@@ -7,6 +7,7 @@ ROOTTEST_ADD_TEST(countIncludePaths
 ROOTTEST_GENERATE_REFLEX_DICTIONARY(expressiveErrorMessages expressiveErrorMessages.h SELECTION expressiveErrorMessages_selection.xml)
 
 ROOTTEST_ADD_TEST(expressiveErrorMessages
+                  REQUIRES pch
```

Comment:
This is not the reading and the translation I'd immediately think of. OTOH, `ENABLE_IF pch`, reads very awkward to me.

Reply:
I like `ENABLE_IF` for a different reason: it's a clearer description of the feature, especially when taking into account `DEPENDS` which we also have.

Reply:
> What would be the negative form ENABLE_IF_NOT?

ENABLE_UNLESS ?

> ENABLE_IF pch

Well the idea is to have a short form instead of 

TURN_TEST_ON_IF_FOLLOWING_FEATURE_ARE_ENABLED  pch

:)

Ah maybe

ENABLE_ONLY_WITH

or

USE_ONLY_IF

or something else better?